initfiles/componentfiles/configxml/@temp/esp_service_WsSMC.xsl

```diff
@@ -696,6 +696,22 @@ This is required by its binding with ESP service '<xsl:value-of select="$espServ
                      </Feature>
                   </xsl:if>
                </xsl:for-each>
+            </xsl:if>
+
+            <xsl:for-each select="$bindingNode/AuthenticateSetting[@include='Yes']">
```

Comment:
Hmmmm.  It's fine for ConfigMgr generated environments, but can someone editing by hand think they are adding security info, but it won't be picked up if they use the wrong case?

It would be common to type all lower, and if we just fail gracefully no one might realize the mistake.

Reply:
but if that's how it is for other types, then might as well just be consistent for now.  So Ok.